pkg/driver/node.go

```diff
@@ -44,6 +44,10 @@ const (
 
 	// default file system type to be used when it is not provided
 	defaultFsType = FSTypeExt4
+
+	// maxVolumesPerNode is the maximum number of volumes that an AWS instance can have attached.
+	// More info at https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html
+	maxVolumesPerNode = 39
```

Comment:
Since NVME volume only allow 28 attachment, and some instance type has EBS as default volume. Feels its safer to set the value to 27 to begin with to accommodate worse case.

Reply:
I added some logic to return the volumes limit based on the instance type.

Reply:
PTAL and let me know if it makes sense.